src/pages/Researchers/ProgramPostForm.js

```diff
@@ -75,19 +75,20 @@ const ProgramPostForm = () => {
       console.log(values);
       console.log(selectedTags);
 
-      await Db.addProgram(userId, values, selectedTags).then;
+      await Db.addProgram(userId, values, selectedTags);
```

Comment:
Accidentally left an extra .then here in the previous PR, sorry!

Reply:
ah, should have caught that, my bad!